gcp/table_gcp_compute_instance_template.go

```diff
@@ -156,6 +156,12 @@ func tableGcpComputeInstanceTemplate(ctx context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 				Transform:   transform.FromField("Properties.Tags"),
 			},
+			{
+				Name:        "labels",
+				Description: "Labels associated with the instance template.",
```

Comment:
How are labels used with instance templates? From the API I can see the description:
`Labels to apply to instances that are created from these properties.`

So are these labels on the instance template themselves and/or labels applied to instances created from these templates?

Also, description wise, similar to the comment below, where did you get this comment from? The API has a different description (https://cloud.google.com/compute/docs/reference/rest/v1/instanceTemplates).